pages/common/git.md

```diff
@@ -14,6 +14,10 @@
 
 `git help {{command}}`
 
+- Call tldr on a command:
+
+`tldr {{git-command}}`
```

Comment:
There should not be a tldr command in a git page.

Reply:
I understand, but when a user search for help with `git checkout`, how he can find the result?
If you say that he gets the result from `tldr tldr`, I will accept.

Reply:
$ tldr git-checkout

git checkout

Checkout a branch or paths to the working tree.
...

Although most clients automatically add a "-" between words.

Reply:
@Accurate0 , I know this now, but

> I searched for a record for a `git checkout` and nearly opened an issue on it.
> I think if there is a user who is less engaged in open source, he would not know how to get records for subcommands.

Just after traveling in this repo, I found the solution. So if there is a user that not familiar with Github or open-source, how he would find the solution?

Reply:
The most basic tendency is to write the exact command; which in this case is "git checkout". So if you do "tldr git checkout", it should give the correct result. And the node client indeed does that. So I don't see any confusion here.

Now, if some client does not implement this feature, and the require you to pass a `-` or whatever, I would expect the person to file an issue with the appropriate client repo asking what the solution is.

Please note that this repo is just for pages, and we do not control the client authors. There are lots of clients in lots of programming languages. If you have any confusion regarding how to find a command, you should ask that client's owner.

Does that clarify things ?

Reply:
Yes, Thank you